### ISSUE TITLE
PhraseApp::RequestParams::LocaleFileImportParams treats TrueClass as falsey

### ISSUE THREAD

**Issue by yob** [phrase/phraseapp-ruby#7]
While debugging #6, I noticed something odd. I was attempting to upload a YAML file, and I want to skip upload tags, so I tried something like this:

```ruby
params = PhraseApp::RequestParams::LocaleFileImportParams.new(
  file: "tmp/test.yml",
  file_format: "yml",
  tags:  "testing",
  skip_upload_tags: true,
)
res = PhraseApp::Client.upload_create(@project_id, params)
```

However, when I inspected the body that is submitted to the API I noticed that skip_upload_tags was set to false:

```
--{PHRASE!!EOF}
Content-Disposition: form-data; name="file"; filename="test.yml"
Content-Type: text/plain

en:
  foo:
    bar: "hi"

--{PHRASE!!EOF}
Content-Disposition: form-data; name="file_format"

yml
--{PHRASE!!EOF}
Content-Disposition: form-data; name="skip_upload_tags"

false
--{PHRASE!!EOF}
Content-Disposition: form-data; name="tags"

testing
```

It's quite surprising to see `true` (a TrueClass) treated as falsey. I found that if I changed the code to specify `skip_upload_tags: "true"` (a String) then the request body has the appropriate value.

Now that I know the trick I can continue using strings to specify a true value, but it'd be much less surprising for new users if `TrueClass` was truthey too.

**Comment by kevvvvv** [comment]
Thank your for pointing this out @yob - we will fix this in a upcoming release.

**Comment by kevvvvv** [comment]
Hi @yob, this behaviour is fixed in the latest release, version 1.0.20.
